instrumentation/redisson/redisson-common/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/redisson/RedissonRequest.java

```diff
@@ -48,37 +44,30 @@ public String getOperation() {
       CommandsData commandsData = (CommandsData) command;
       if (commandsData.getCommands().size() == 1) {
         return commandsData.getCommands().get(0).getCommand().getName();
+      } else {
+        return "BATCH EXECUTE";
       }
     }
     return null;
   }
 
   @Nullable
   public String getStatement() {
-    List<String> sanitizedStatements = sanitizeStatement();
-    switch (sanitizedStatements.size()) {
-      case 0:
-        return null;
-        // optimize for the most common case
-      case 1:
-        return sanitizedStatements.get(0);
-      default:
-        return String.join(";", sanitizedStatements);
-    }
+    return sanitizeStatement();
```

Comment:
instead of omitting the statement when multiple statements are executed you could have changed it to only capture some small number X statements or limit the total length of the statement (there is a size limit to attributes anyway). Did you consider solving it this way?